crates/bevy_transform/src/components/children.rs

```diff
@@ -29,6 +31,35 @@ impl Children {
     pub fn swap(&mut self, a_index: usize, b_index: usize) {
         self.0.swap(a_index, b_index);
     }
+
+    /// Returns result of the subquery filtered by the children
```

Comment:
This isn't returning `query.iter().filter(|entity| children.contains(entity))` as this comment would suggest, it returns an iterator over the children mapped and filtered on the query.

Reply:
I rephrased it to

> Returns an iterator over the query result for each child